demucs/apply.py

```diff
@@ -208,30 +220,44 @@ def apply_model(model, mix, shifts=1, split=True,
         # We start from a triangle shaped weight, with maximal weight in the middle
         # of the segment. Then we normalize and take to the power `transition_power`.
         # Large values of transition power will lead to sharper transitions.
-        weight = th.cat([th.arange(1, segment // 2 + 1, device=device),
-                         th.arange(segment - segment // 2, 0, -1, device=device)])
+        weight = th.cat(
+            [
+                th.arange(1, segment // 2 + 1, device=device),
+                th.arange(segment - segment // 2, 0, -1, device=device),
+            ]
+        )
         assert len(weight) == segment
         # If the overlap < 50%, this will translate to linear transition when
         # transition_power is 1.
-        weight = (weight / weight.max())**transition_power
+        weight = (weight / weight.max()) ** transition_power
         futures = []
         for offset in offsets:
             chunk = TensorChunk(mix, offset, segment)
             future = pool.submit(apply_model, model, chunk, **kwargs)
             futures.append((future, offset))
             offset += segment
         if progress:
-            futures = tqdm.tqdm(futures, unit_scale=scale, ncols=120, unit='seconds')
+            futures = tqdm.tqdm(
+                futures,
+                unit_scale=scale,
+                ncols=75,
+                unit="s",
+                bar_format="{desc}: {percentage:3.0f}%|{bar}|{n:.0f}/{total:.0f} [{elapsed}<{remaining},{rate_fmt}{postfix}]",
```

Comment:
Please don’t change the `unit` to s. This will make it output things like `5s/s`, which can be understood as `5seconds/s` or `0.2seconds/s` as tqdm ensures the number is no less than 1.